src/base58.cpp

```diff
@@ -12,6 +12,24 @@
 
 /** All alphanumeric characters except for "0", "I", "O", and "l" */
 static const char* pszBase58 = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
+static const int8_t mapBase58[] = {
```

Comment:
If you make this
```
static const int8_t mapBase58[256] = {
```
That's pretty much a static assertion that the size will be 256.